src/stores/configuration.md

```diff
@@ -18,5 +18,5 @@ In this guide, you can drill down through any section of the reference and follo
 - [Services]({% link configuration/services.md %})
 - [Advanced]({% link configuration/advanced.md %})
 
-![Magento Admin - Configuration]({% link images/images-ee/config-ee.png %}){: .zoom}
-_Configuration_{:.ee-only}
+![Magento Admin - Configuration]({% link images/images/config.png %}){: .zoom}
```

Comment:
Hi @EvheniiaKolkhiieva this should be:

![Magento Admin - Configuration]({% link images/images/configuration.png %}){: .zoom}

correct?, I just noticed this after @dobooth reviewed it, thanks!

Reply:
Hi @danidelcar, thank you for the review 😊 I have added a new screenshot to the images/images folder, so I think this is the correct display. 
But if you about structure, - the similar you can see in the other places 
![image](https://user-images.githubusercontent.com/78729171/118131718-89868f80-b407-11eb-804c-1d2158d3e685.png)
@dobooth please correct me if I'm wrong!

Thank you!

Reply:
Hi @EvheniiaKolkhiieva , the png name should be **configuration.png** so is pointing to the new image you added, correct?. Instead of **config.png** in the link

Reply:
Oh, that's what you mean @danidelcar , yes you are absolutely right, excellent observation!
I fixed this mistake. Thank you a lot!

Reply:
Thanks @dobooth ! 😅